bindings/responses.go

```diff
@@ -19,6 +19,7 @@ type ReadResponse struct {
 type AppResponse struct {
 	Data        interface{}        `json:"data"`
 	To          []string           `json:"to"`
-	State       []state.SetRequest `json:"state"`
+	StoreName   string             `json:"storeName"`
+	Requests    []state.SetRequest `json:"requests"`
```

Comment:
There is no reason to change this, right? Also, this is seems to be a response object, so calling it requests reads odd (IMO). Can you just add the State attribute and be done?

Reply:
Though this is app response, but that can have state request to store after binding event. 
I did not get the State attribute part as how to achieve through the attribute..